src/Tester.java

```diff
@@ -0,0 +1,16 @@
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Tester {
+
+    public static void main(String[] args) {
+
+        Stream.iterate(8, x -> x + 1)
```

Comment:
Nadawaj lepsze nazwy zmiennych, po prostu dla czytelności. Daje to możliwość domyślenia się po samej nazwie czym jest aktualnie przetwarzany element w danej fazie strumienia.